src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,117 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkBase;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.*;
+import frc.lib.Gains;
+import frc.lib.Neo;
+
+import java.util.function.DoubleSupplier;
+import java.util.function.Supplier;
+
+import static frc.robot.Constants.ClimberConstants.*;
+import static frc.robot.util.AllianceUtils.isBlueAlliance;
+
+public class Climber extends SubsystemBase {
+    ClimberSide leftSide;
+    ClimberSide rightSide;
```

Comment:
Don't leave fields package-private. Like @TapChap reviewed earlier, these should be private final.